### ISSUE TITLE
Create release 0.3.0

### ISSUE THREAD

**Issue by Ainali** [publiccodenet/standard#544]
Since we have a meaningful clarification in (and splitting of) a MUST requirement (in the criteria Use plain English), I believe it would be proper to bump the version to 0.3.0.

**Comment by ericherman** [comment]
Given the change:

> -* All code and documentation MUST be in English.
> +* All codebase documentation MUST be in plain English.
> +* All code MUST be in English, except where required to execute as policy.

If the original requirement was fulfilled, then both halves of were fulfilled, which is now expressed as two distinct requirements. One of which is slightly looser, yet one could argue that the switch from "English" to "plain English" more strict.

Upon closer inspection, the SHOULD requirements of the section describe plain English, which creates friction with the new definition in the glossary.

My sense here is that we probably do not want "plain" in the new split requirement, which would make this more of a patch, than a change in requirements.

**Comment by ericherman** [comment]
See: https://github.com/publiccodenet/standard/pull/545

**Comment by Ainali** [comment]
I agree, let's make a new issue for another release.
